### PR TITLE
lesson_2 done

### PR DESCRIPTION
**Pull request by AlexRubenko**
Lesson_2 done (Flask views).

### CODE REVIEW

File: Lesson_2/Lesson_2.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+@app.route('/calculate_average')
+def calculate_average():
+    filename = 'hw.csv'
+    with open(filename) as csv_file:
+        reader = csv.reader(csv_file)
+        next(reader)  # пропустить заголовок
+        col_1_sum = 0
+        col_2_sum = 0
```
Comment (mikolaz27) [inline Lesson_2/Lesson_2.py:36]: rename variables. Dont use numbers in vars

File: Lesson_2/Lesson_2.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    return ''.join(choices(string.ascii_lowercase + string.ascii_uppercase + string.punctuation + string.digits,
+                           k=password_length))
+
+
+@app.route('/calculate_average')
+def calculate_average():
+    filename = 'hw.csv'
+    with open(filename) as csv_file:
+        reader = csv.reader(csv_file)
+        next(reader)  # пропустить заголовок
```
Comment (mikolaz27) [inline Lesson_2/Lesson_2.py:34]: English comments or remove

File: Lesson_2/Lesson_2.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        col_2_sum = 0
+        count = 0
+        for row in reader:
+            if len(row) >= 3:
+                col_1_sum += float(row[1])
+                col_2_sum += float(row[2])
+                count += 1
+        if count > 0:
+            average_height = (col_1_sum / count) * 2.54
+            average_weight = (col_2_sum / count) * 0.453
```
Comment (mikolaz27) [inline Lesson_2/Lesson_2.py:45]: What is this number?  2.54 and 0.453

File: Lesson_2/Lesson_2.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        next(reader)  # пропустить заголовок
+        col_1_sum = 0
+        col_2_sum = 0
+        count = 0
+        for row in reader:
+            if len(row) >= 3:
+                col_1_sum += float(row[1])
+                col_2_sum += float(row[2])
+                count += 1
+        if count > 0:
```
Comment (mikolaz27) [inline Lesson_2/Lesson_2.py:43]: Check pandas you can make it easier.

File: Lesson_2/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Package      Version
```
Comment (mikolaz27) [inline Lesson_2/requirements.txt:1]: use pip freeze

File: Lesson_2/Lesson_2.py (hunk 3 above)
Comment (AlexRubenko) [reply Lesson_2/Lesson_2.py:45]: These are conversion factors for length from inches and mass from pounds to the SI system.

**Pull request merged**